src/main/java/mate/academy/mainspringproject/model/Order.java

```diff
@@ -0,0 +1,58 @@
+package mate.academy.mainspringproject.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+
+@Entity
+@Data
+@Table(name = "orders")
+@SQLDelete(sql = "UPDATE orders SET is_deleted = TRUE WHERE id = ?")
+@SQLRestriction("is_deleted = FALSE")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
+    @Column(nullable = false)
+    @Enumerated(EnumType.STRING)
+    private Status status;
+    @Column(nullable = false)
+    private BigDecimal total;
+    @Column(name = "order_date", nullable = false)
+    private LocalDateTime orderDate;
+    @Column(name = "shipping_address", nullable = false)
+    private String shippingAddress;
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    @Column(name = "order_items")
+    @OneToMany (mappedBy = "order")
+    private Set<OrderItem> orderItems = new HashSet<>();
+    @Column(name = "is_deleted")
```

Comment:
this changes nothing
```suggestion
    @Column(nullable = false)
    private LocalDateTime orderDate;
    @Column(nullable = false)
    private String shippingAddress;
    @ToString.Exclude
    @EqualsAndHashCode.Exclude
    @OneToMany (mappedBy = "order")
    private Set<OrderItem> orderItems = new HashSet<>();
```